doc/ref/api/index.md

```diff
@@ -17,6 +17,17 @@
    show
 ```
 
+## Sample Data
+
+The [sampledata](manual/sampledata.md) module provides convenient access to sample datasets for learning and prototyping.
+
+```{toctree}
```

Comment:
Instead of having the `sampledata.md` file, can't we add just a bit of text in this section? Really, I don't want to have to maintain hvsampledata's documentation in hvPlot, it should be all in its own website.

Reply:
OK